app/routers/sessions.py

```diff
@@ -25,25 +25,39 @@ def str_to_datetime(datetime_str: str) -> datetime:
 
 @router.post("/", response_model=SessionResponse)
 async def create_session(session: Session):
-    session = jsonable_encoder(session)
+    current_session = jsonable_encoder(session)
 
-    quiz = client.quiz.quizzes.find_one({"_id": session["quiz_id"]})
+    quiz = client.quiz.quizzes.find_one({"_id": current_session["quiz_id"]})
 
     if quiz is None:
         raise HTTPException(
-            status_code=404, detail=f"quiz {session['quiz_id']} not found"
+            status_code=404, detail=f"quiz {current_session['quiz_id']} not found"
         )
 
-    previous_session = client.quiz.sessions.find_one(
-        {"quiz_id": session["quiz_id"], "user_id": session["user_id"]},
-        sort=[("_id", pymongo.DESCENDING)],
+    # try to get the previous two sessions of a user+quiz pair if they exist
+    previous_two_sessions = list(
+        client.quiz.sessions.find(
+            {
+                "quiz_id": current_session["quiz_id"],
+                "user_id": current_session["user_id"],
+            },
+            sort=[("_id", pymongo.DESCENDING)],
+            limit=2,
+        )
     )
+    last_session, second_last_session = None, None
+    if len(previous_two_sessions) == 1:
+        # only one session exists
```

Comment:
can you place this comment and the below comment just above the `if` and `elif` statements?